controllers/ipfscluster_controller.go

```diff
@@ -233,3 +227,22 @@ func (r *IpfsClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			MaxConcurrentReconciles: 1,
 		}).Complete(r)
 }
+
+func getBootstrapAddrs(secret *corev1.Secret, relayPeers []peer.AddrInfo) ([]string, error) {
+	bootstrapPeers := []string{}
+	peer0IDKey := KeyPeerIDPrefix + "0"
+	peer0ID, ok := secret.Data[peer0IDKey]
+	if !ok {
+		return nil, fmt.Errorf("could not retrieve initial peer id")
+	}
+	initPeerID := string(peer0ID)
+	// construct a p2p circuit here
+	for _, peer := range relayPeers {
+		circuitID := peer.ID
+		for _, circuitAddr := range peer.Addrs {
+			bootstrapAddr := fmt.Sprintf("%s/p2p/%s/p2p-circuit/p2p/%s", circuitAddr.String(), circuitID.String(), initPeerID)
+			bootstrapPeers = append(bootstrapPeers, bootstrapAddr)
```

Comment:
This seems off... peers in a private network would not need to use relays to contact each other as they can dial each other directly over their LAN IPs...

There's also a question of what are relays for in private networks. The relays cannot communicate to ipfs nodes at all if those nodes are configured with a PNET key. The relays would need support for private networks too.